formPage/my-app/src/CheckinForm.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+async function getCheckin(state) {
+    const request = await fetch("http://localhost:5000/updateCheckin", {
```

Comment:
You can make API calls (POST, GET, PUT, etc) functions that take in an URL and body so you can avoid re-writing this boiler plate code (ie. https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)

Also, does the API work here if you don't supply cors?